app/features/project/project_controller.rb

```diff
@@ -66,21 +67,20 @@ class ProjectController < AuthenticatedControllerBase
         clone_url: project.repo_config.git_url
         # we don't need to pass a `ref`, as the sha and branch is all we need
       )
+      trigger = project.job_triggers.find do |t|
+        t.type == FastlaneCI::JobTrigger::TRIGGER_TYPE[:manual]
+      end
 
-      build_runner = FastlaneBuildRunner.new(
+      remote_runner = RemoteRunner.new(
```

Comment:
Can we update the complete code base to use `RemoteRunner` instead of `FastlaneBuild` runner? 👍

Reply:
We could but I'm not sure if we should do that. 
I can prepare another commit to update the controller used by the new frontend to use the Agent for the manually triggered builds, but I was thinking we might want to see things merged and running for the manual trigger before updating everything?

Reply:
Updated the BuildJSONController to use the RemoteRunner for the manual trigger. 
Once we merge this, we can try in production for manual triggering and if all looks alright we can replace `FastlaneBuildRunner` for the other triggers.